chapters/curate-maintain/community/reading/read.md

```diff
@@ -1,10 +1,28 @@
 # Community
 
-Owner: Răzvan Deaconescu
-Reviewer: Răzvan Nițu
+The use of [communication](../../communication/reading/read.md) media around the content repository is both to build and to maintain a healthy community of educators and learners around the topic of the repository.
+The community will serve multiple roles:
 
-building a community around the project, encouraging [communication](../../communication/reading/read.md)
+- Provide feedback to existing content, new content and to decisions enacted by the content curator and other leaders.
+- Submit contributions and do reviews and enrich existing content.
+- Take part in the decision making process.
+- Provide new ideas for new content, new infrastructure and other support items.
 
-community of educators and learners
+The community will include all types of [roles](../../../roles/overview/reading/read.md).
+The content curator will act as a community leader that will involve as many members, particularly experienced and active contributors, content developers and educators, in the decision making process.
 
-advertising content to potential users
+The core of the community will consist of educators, learners and contributors / hobbyists.
+
+Is is the goal and the duty of the content curator to engage community members and maintain a vibrant environment.
+
+The content curator is to consistently look to enlarging the community.
+To this end, the content curator must periodically take the following actions:
+
+- Advertise the content repositories on social media and other communication media.
+- Take part in events, talks, conferences, meetups and hackathons to present the community, the content and the infrastructure.
+- Maintain an active connection to educators that use the content, collect feedback and aim to improve it to further the use of the content in educator institutions.
+- Engage contributors and reviewers for an effective and timely review process, to stimulate consistent submissions.
+- Look for sponsors and funding for content developers and reviewers.
+
+The content curator is to set up a "council" of the community, to assist with the decision making process and share the responsibilities in leading the community.
+This is to be organized by the content curator, with recommendations on attracted experienced educators and contributors and those who show interest in taking a more active role.
```

Comment:
```suggestion
This is to be organized by the content curator, with recommendations on attracting experienced educators and contributors and those who show interest in taking a more active role.
```
or "recommendations **from** attracted blah blah"; not exactly sure what you're trying to say here.